TextClassifier.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bolt TextClassifier Demo\n",
+    "\n",
+    "The bolt TextClassifier is an easy way to quickly develop, train, and test a model to perform classification and categorization of text data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from thirdai import bolt\n",
+    "import datasets\n",
+    "import random\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define Helper Functions \n",
+    "def write_dataset_to_csv(dataset, filename, return_labels=False):\n",
+    "    label_names = dataset.features[\"intent\"].names\n",
+    "\n",
+    "    data = []\n",
+    "    for item in dataset:\n",
+    "        sentence = item[\"text\"]\n",
+    "        label = item[\"intent\"]\n",
+    "        label_name = label_names[label]\n",
+    "        data.append((sentence, label_name))\n",
+    "\n",
+    "    random.shuffle(data)\n",
+    "\n",
+    "    with open(filename, \"w\") as file:\n",
+    "        file.write('\"text\",\"category\"\\n')\n",
+    "        lines = [\n",
+    "            '\"{}\",\"{}\"\\n'.format(sentence, label_name) for sentence, label_name in data\n",
```

Comment:
Same comment as in the other PR about potentially switching to f-strings. It's up to you but I think it's nicer to read.